pkg/driver/node.go

```diff
@@ -165,6 +165,12 @@ func (d *Driver) NodePublishVolume(ctx context.Context, req *csi.NodePublishVolu
 				}
 			}
 
+			if f == "awscredsuri" {
```

Comment:
1. I would suggest to have a non supported options list, and check whether that option is in that list
2. If that option is not supported, I think we should fail gracefully by just ignore that option + warn log, instead of return an error

Reply:
thanks, logged this behavior, return nil but do not fail

Updated this behavior in ReadMe